packages/components/src/internal/components/editable/LookupCell.tsx

```diff
@@ -92,29 +93,29 @@ export class LookupCell extends PureComponent<LookupCellProps> {
             );
         }
 
-        if (lookup.hasQueryFilters()) {
-            queryFilters = queryFilters.push(...lookup.getQueryFilters());
+        const operation = forUpdate ? 'update' : 'insert';
```

Comment:
Should we consider enumerating these operations? `Operation.update`, `Operation.insert`? Are there others (I haven't reviewed server-side implementation yet)?